src/core/search/search.cc

```diff
@@ -56,33 +110,60 @@ struct BasicSearch {
     return casted_ptr;
   }
 
-  vector<DocId> Search(monostate, string_view) {
-    return {};
+  // Collect all index results from F(C[i]) and sort by size
```

Comment:
you do not sort here, though.

Reply:
yes, forgot to update the comment, moved the sort to the function below